packages/cli/src/lib/new/factories/backendPlugin.ts

```diff
@@ -78,6 +78,31 @@ export const backendPlugin = createFactory<Options>({
           },
         );
       });
+
+      await Task.forItem('backend', 'adding plugin', async () => {
+        const backendFilePath = paths.resolveTargetRoot(
+          'packages/backend/src/index.ts',
+        );
+        if (!(await fs.pathExists(backendFilePath))) {
+          return;
+        }
+
+        const content = await fs.readFile(backendFilePath, 'utf8');
+        const lines = content.split('\n');
+        const backendAddLine = `backend.add(import('${name}'));`;
+
+        const backendStartIndex = lines.findIndex(line =>
+          line.match(/backend.start/),
+        );
+
+        if (backendStartIndex !== -1) {
+          const [indentation] = lines[backendStartIndex].match(/^\s*/) ?? [];
```

Comment:
```suggestion
          const [indentation] = lines[backendStartIndex].match(/^\s*/)!;
```

i mean it just will never fail - that solves the `| undefined` problem right?